common/src/main/java/com/wynntils/models/npc/label/NpcLabelParser.java

```diff
@@ -32,14 +35,18 @@ public NpcLabelInfo getInfo(StyledText label, Location location, Entity entity)
                     matcher.group("description"));
         }
 
-        if (label.matches(TRADE_MARKET_LABEL_PATTERN)) {
-            return new NpcLabelInfo(label, "Trade Market", location.offset(0, -1, 0), entity);
-        }
-
         if (label.matches(HOUSING_LABEL_PATTERN)) {
             return new NpcLabelInfo(label, "Housing", location, entity);
         }
 
+        if (label.matches(BOOTH_SHOP_LABEL_PATTERN)) {
+            return new NpcLabelInfo(label, "Booth Shop", location.offset(0, -1, 0), entity);
+        }
+
+        if (label.matches(SEASKIPPER_LABEL_PATTERN)) {
+            return new NpcLabelInfo(label, "Seaskipper", location.offset(0, -1, 0), entity);
+        }
+
```

Comment:
The more I see this, the more I think we should do this correction on post processing, but the files are versioned anyways, so it's not a huge deal.